index.html

```diff
@@ -0,0 +1,262 @@
+                        <img id="counter">
+                        <section id="role">
+                            <p id="pcareer">Back End Dev</p>
+                        </section>
+                        <img id="counter">
+                        <section id="year_label">
+                            <p id="pcareer">2015</p>
+                        </section>
+                    </section>
+                </section>
+                <p id="sub_text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <section id="tags">
+                    <section id="tag1">
+                        <p id="thtml">html</p>
+                    </section>
+                    <section id="tag2">
+                        <p id="tcss">css</p>
+                    </section>
+                    <section id="tag3">
+                        <p id="tjavascript">javascript</p>
+                    </section>
+                </section>
+                <div id="action">
+                    <button id="normal_button">
+                        <p id="see_project">See Project</p>
+                    </button>
+                </div>
+            </section>
+        </section>
+        <section id="card_works">
+            <img id="snapshoot_portfolio" src="images/SnapshootPortfolio(3).svg" alt="SnapshootPortfolio picture">
+            <section id="left_block">
+                <section id="primary_text">
+                    <p id="project_title">Multi-Post Stories</p>
+                    <section id="frame2">
+                        <section id="client">
+                            <p id="pcanopy">CANOPY</p>
+                        </section>
+                        <img id="counter">
+                        <section id="role">
+                            <p id="pcareer">Back End Dev</p>
+                        </section>
+                        <img id="counter">
+                        <section id="year_label">
+                            <p id="pcareer">2015</p>
+                        </section>
+                    </section>
+                </section>
+                <p id="sub_text">A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <section id="tags">
+                    <section id="tag1">
+                        <p id="thtml">html</p>
+                    </section>
+                    <section id="tag2">
+                        <p id="tcss">css</p>
+                    </section>
+                    <section id="tag3">
+                        <p id="tjavascript">javascript</p>
+                    </section>
+                </section>
+                <div id="action">
+                    <button id="normal_button">
+                        <p id="see_project">See Project</p>
+                    </button>
+                </div>
+            </section>
+        </section>
```

Comment:
- [ ] Please note that all your cards should be in one section. You have used so many sections in a section that it's better to use your cards, which can be articles in a section or just divs in a section. Also, you are to align the contents of the section (that is, the cards) using a CSS grid as demanded by project requirements. Kindly fix this, please.